k8s/namespaces/rd/rd-caseworker-ref-api/demo.yml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: rd-caseworker-ref-api
+  annotations:
+    fluxcd.io/automated: "true"
+    fluxcd.io/tag.java: glob:demo-*
```

Comment:
This doesn't work, u need to point to a PR.
Also, this should be added to kustomize